bats/tests/containers/factory-reset.bats

```diff
@@ -32,13 +32,7 @@ start_application() {
         wait_for_container_engine
     fi
 
-    # BUG BUG BUG
-    # Looks like the rcfiles don't get updated via `rdctl start`
-    # BUG BUG BUG
-    if is_unix; then
-        rdctl set --application.path-management-strategy manual
-        rdctl set --application.path-management-strategy rcfiles
-    fi
+    ensure_dotfiles_are_completed_BUG_BUG_BUG_4519
```

Comment:
I wanted to move this code to `start_container_engine()` in `helpers/vm.bash` instead of calling it from individual tests, but I don't really care too much, as we are going to remove the workaround anyways, as soon as we have fixed the underlying issue.

Reply:
The idea behind `wordarounds.bash` is that it contains temporary code and should normally be empty